satellite/audit/reservoir.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"math/rand"
+
+	"storj.io/storj/pkg/storj"
+)
+
+const maxReservoirSize = 2
+
+// Reservoir holds a certain number of segments to reflect a random sample
+type Reservoir struct {
+	Paths [maxReservoirSize]storj.Path
+	size  int8
+	index int64
+}
+
+// NewReservoir instantiates a Reservoir
+func NewReservoir(size int) *Reservoir {
+	if size > maxReservoirSize {
+		size = maxReservoirSize
+	}
```

Comment:
What if `size < 1`?

Reply:
haha honestly I'm not sure how to handle. I guess I could just set it to maxReservoirSize too? so then the size will always be either 1 or 2 and the configurer will have the illusion of choice

Reply:
If below 1 then it would be better to set to 1 than to the max.